packages/near-api-js/src/account_multisig.ts

```diff
@@ -390,7 +390,7 @@ export class Account2FA extends AccountMultisig {
         return requestId;
     }
 
-    async getCodeDefault(method: any): Promise<string> {
+    async getCodeDefault(): Promise<string> {
         throw new Error('There is no getCode callback provided. Please provide your own in AccountMultisig constructor options. It has a parameter method where method.kind is "email" or "phone".');
     }
 
```

Comment:
Why have these types changed?

Reply:
removed the arg since it's unused. This surfaced as part of removing lint errors